gdi/opentelemetry/install-linux.rst

```diff
@@ -48,7 +48,7 @@ To install the package using the installer script, follow these steps:
    curl -sSL https://dl.signalfx.com/splunk-otel-collector.sh > /tmp/splunk-otel-collector.sh;
    sudo sh /tmp/splunk-otel-collector.sh --realm SPLUNK_REALM --memory SPLUNK_MEMORY_TOTAL_MIB -- SPLUNK_ACCESS_TOKEN
 
-.. note:: If you don't have a Log Observer entitlement or don't wish to collect logs for the target host, use the ``--without-fluentd`` option to skip the installation of Fluentd when installing the Collector.
+.. note:: If you have a Log Observer entitlement or wish to collect logs for the target host with Fluentd, use the ``--with-fluentd`` option to also install Fluentd when installing the Collector.
```

Comment:
@jeffreyc-splunk Would you require Fluentd when sending logs for LO? Wouldn't filelog receiver also work?

Reply:
> @jeffreyc-splunk Would you require Fluentd when sending logs for LO? Wouldn't filelog receiver also work?

I just updated this line for the inverse.  I guess the filelog receiver would work, but we currently don't include it in our default collector configs.